sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ServiceApiTests.java

```diff
@@ -658,6 +658,25 @@ public void setPropsMin() {
         }
     }
 
+    private static Stream<Arguments> setAndGetServiceVersionSupplier() {
+        return Stream.of(
+            Arguments.of(BlobServiceVersion.V2020_02_10, BlobServiceVersion.V2020_02_10)//,
+            //Arguments.of(null, BlobServiceVersion.getLatest())
```

Comment:
Was this intended to be commented out? I'm also fine just having these as two separate test cases if trying to parameterize and handle a service version of `null` muddies the test case's logic.